src/editor.rs

```diff
@@ -5,12 +5,24 @@ use crossterm::{
     terminal::{self, ClearType},
     QueueableCommand, Result,
 };
+use serde::{Deserialize, Serialize};
 use std::io::{stdout, Write};
 
 use crate::{fred_file, term};
 
 const TABASSPACES: u16 = 4;
```

Comment:
If we are using the config value now, we no longer need this variable - `TABSPACES`